libs/category/state/src/effects/category-page.effects.ts

```diff
@@ -75,10 +76,9 @@ export class DaffCategoryPageEffects<
     withLatestFrom(
       this.store.pipe(select(this.categorySelectors.selectCategoryPageMetadata)),
     ),
-    switchMap((
-      [action, categoryRequest]: [DaffCategoryPageChangePageSize, DaffCategoryRequest],
-    ) => this.processCategoryGetRequest({
+    switchMap(([action, categoryRequest]: [DaffCategoryPageChangePageSize, DaffCategoryPageMetadata]) => this.processCategoryGetRequest({
       ...categoryRequest,
+      kind: DaffCategoryPageRequestKind.ID,
```

Comment:
Why is the `kind` being spread in here shouldn't this be on the request?

Reply:
that's the metadata being spread, not actually the request. I improving its name though